build.gradle

```diff
@@ -1,38 +1,56 @@
-plugins {
-    id 'java'
+// First apply all plugins
+
+apply plugin: "java"
+apply plugin: "com.google.protobuf"
+apply plugin: "idea"
+
+repositories {
+    mavenCentral()
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
+    }
 }
 
-group 'neonracer'
-version '0.1.0'
+group "neonracer"
+version "0.1.0"
 
 sourceCompatibility = 1.8
 
 project.ext.mainClass = "neonracer.NeonRacerMain"
```

Comment:
```suggestion
```
Gehört zwar nicht dazu, aber die Zeile hier kann eigentlich raus

Reply:
Wenn ich diese Zeile entferne kompiliert bei mir das Projekt nicht mehr: `Could not get unknown property 'mainClass' for object of type org.gradle.api.java.archives.internal.DefaultManifest.`

Reply:
Ach stimmt wegen dem jar output. Ich dachte das wäre nur für die alte Startmethode gewesen. Hab ganz vergessen, dass wir noch einen Jar Out drin haben. Gut dann lass es drin.